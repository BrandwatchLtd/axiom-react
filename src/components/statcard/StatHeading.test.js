import React from 'react';
import renderer from 'react-test-renderer';
import { StatHeading } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <StatHeading { ...props }>Lorem</StatHeading>
  );
}

describe('StatHeading', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
