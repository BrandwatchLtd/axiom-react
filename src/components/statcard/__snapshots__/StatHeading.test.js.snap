// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatHeading renders with defaultProps 1`] = `
<div
  className="ax-stat-cards__heading"
>
  Lorem
</div>
`;
